### DISCUSSION TITLE
deploy Yaml files from different folders

### DISCUSSION THREAD

**Discussion by arnitolog** [argoproj/argo-cd#7585] [General]
Hello, I there a way to deploy yaml files from different locations in the same ArgoCD app?
For example, imagine the below structure:

Dev:
 |_ monitor1.yaml
 |_ monitor2.yaml
Prod:
 |_ prod_monitor1.yaml
 |_ prod_monitor2.yaml
 ....
 |__ prod_monitorN.yaml
shared:
 |_ shared_monitor1.yaml
 |_ shared_monitor2.yaml

What I want is to deploy everything from the `shared` folder, and depending on the destination deploy everything from `Dev` or `Prod` folders.

**Comment by peana** [comment]
Kustomize may fulfill your needs
https://github.com/kubernetes-sigs/kustomize

**Comment by arnitolog** [comment]
Thank you @peana, kustomize was the first place we looked at.
But unfortunately, it doesn't support wildcards in folders/filenames. At the same time, the number of files and their names in folders Dev/Prod may various.
